features/full_tests/threaded_startup.feature

```diff
@@ -6,4 +6,5 @@ Feature: Switching automatic error detection on/off for Unity
   Scenario: Starting Bugsnag & calling it on separate threads
     When I run "MultiThreadedStartupScenario" and relaunch the crashed app
     And I configure Bugsnag for "MultiThreadedStartupScenario"
-    Then I should receive no error
+    Then I wait to receive an error
+    And the error is correct for "MultiThreadedStartupScenario" or I allow a retry
```

Comment:
Is this the best approach here? If it receives a request it shouldn't that indicates an actual issue.